@here/olp-sdk-dataservice-read/lib/client/ConfigClient.ts

```diff
@@ -44,22 +44,29 @@ export class ConfigClient {
      * If the schema is not set, the filter returns all of the catalogs to which you have access.
      */
     public async getCatalogs(
-        request: CatalogsRequest
+        request?: CatalogsRequest
     ): Promise<ConfigApi.CatalogsListResult> {
         const requestBuilder = await RequestFactory.create(
             "config",
             this.apiVersion,
             this.settings
         ).catch(error => Promise.reject(error));
-        return ConfigApi.getCatalogs(
-            requestBuilder,
-            request.getSchema()
-                ? {
-                      verbose: "true",
-                      schemaHrn: request.getSchema(),
-                      billingTag: request.getBillingTag()
-                  }
-                : { billingTag: request.getBillingTag() }
-        );
+
+        if (request) {
+            return ConfigApi.getCatalogs(
+                requestBuilder,
+                request.getSchema()
+                    ? {
+                          verbose: "true",
+                          schemaHrn: request.getSchema(),
+                          billingTag: request.getBillingTag()
+                      }
+                    : { billingTag: request.getBillingTag() }
+            );
+        } else {
+            return ConfigApi.getCatalogs(requestBuilder, {
+                verbose: "true"
+            });
+        }
```

Comment:
Please return ConfigApi.getCatalogs(requestBuilder, params); and build params depends of conditions